### PR TITLE
Update stale bot message

### LINKED ISSUE #6446: Update stale bot message
From:

This PR has been automatically marked as stale because it has not had activity in the last 30 days. It will be closed if no further activity occurs within 10 days. Thank you for your contributions.

with
```
This PR has been automatically marked as stale because it has not had activity in the last 30 days.
It will be closed if no further activity (new commits/comments) occurs within 10 days.
Thank you for your contributions.

Community members are welcome to grab these works.
```

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Fix #6446

1. Keep newlines. Replace folded '>' with literal '|'.
2. Add "Community members are welcome to grab these works."

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -22,20 +22,24 @@ exemptMilestones: false
 staleLabel: Review - Abandoned
 
 # Comment to post when marking as stale. Set to `false` to disable
-markComment: >
+markComment: |
```
Comment (SteveL-MSFT) [inline .github/stale.yml]: Why change the markComment symbol?

Comment (markekraus) [reply .github/stale.yml]: I think it's better to keep it folded is better. Accirding to spec, if you want to force a new line in in folded you just need to add a blank line between the test blocks.

Comment (iSazonov) [reply .github/stale.yml]: Fixed.

File: .github/stale.yml [hunk 2]
```diff
@@ -24,18 +24,27 @@ staleLabel: Review - Abandoned
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
   This PR has been automatically marked as stale because it has not had activity in the last 30 days.
+
```
Comment (markekraus) [inline .github/stale.yml]: What I meant was that these should not be on separate lines in the message. Only the final note about being up for grabs should be on a new line.

**Review by markekraus — CHANGES_REQUESTED** [review]

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: .github/stale.yml [hunk 3]
```diff
@@ -25,17 +25,24 @@ staleLabel: Review - Abandoned
 markComment: >
   This PR has been automatically marked as stale because it has not had activity in the last 30 days.
   It will be closed if no further activity occurs within 10 days.
+
   Thank you for your contributions.
 
+
```
Comment (markekraus) [inline .github/stale.yml]: Remove extra line.

**Review by markekraus — CHANGES_REQUESTED** [review]

File: .github/stale.yml (hunk 3 above)
Comment (iSazonov) [reply .github/stale.yml]: GitHub editor :-)

**Review by markekraus — APPROVED** [review]

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**